<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignInWidget</class>
 <widget class="QWidget" name="SignInWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>523</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>523</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>523</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>SignInWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#SignInWidget{border-image: url(:/ZcloudDesk/image/signIn.png);border-radius:8px;}</string>
  </property>
  <widget class="QLabel" name="label_round2">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>260</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>21</width>
     <height>21</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>21</width>
     <height>21</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/ZcloudDesk/image/round.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>218</y>
     <width>34</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/ZcloudDesk/image/coin_tips.png);
color: rgb(255, 255, 255);
font: 10px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p style='line-height:15px'&gt;+100&lt;/p&gt;&lt;p style='line-height:-5px'&gt;云币&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>413</y>
     <width>420</width>
     <height>90</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>420</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>420</width>
     <height>90</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>298</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(51, 51, 51);
font: 14px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>10天</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>218</y>
     <width>34</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/ZcloudDesk/image/coin_tips.png);
color: rgb(255, 255, 255);
font: 10px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p style='line-height:15px'&gt;+30&lt;/p&gt;&lt;p style='line-height:-5px'&gt;云币&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCoin">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>63</y>
     <width>121</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>15</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_round1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>21</width>
     <height>21</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>21</width>
     <height>21</height>
    </size>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#label_round1{border-image: url(:/ZcloudDesk/image/round.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>63</y>
     <width>41</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>15</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>云币：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>248</x>
     <y>298</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(51, 51, 51);
font: 14px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>15天</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>298</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(51, 51, 51);
font: 14px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>5天</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>384</x>
     <y>62</y>
     <width>61</width>
     <height>14</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>14</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>1、每日签到可获得10云币
2、每月累计签到5天、10天、15天、
20天，可分别获得额外奖励10、30、
60、100云币奖励
3、累计签到天数以自然月为周期，
每月初重新计算       </string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolTip{background:rgba(255,253,244,1);border:1px solid rgba(255, 253, 244, 1);color:#333333;font: 12px &quot;微软雅黑&quot;;}
QLabel#label_3{color:rgb(255, 255, 255);font:14px &quot;微软雅黑&quot;;};

</string>
   </property>
   <property name="text">
    <string>签到规则</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_round4">
   <property name="geometry">
    <rect>
     <x>359</x>
     <y>255</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>31</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>31</width>
     <height>31</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/ZcloudDesk/image/gift.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>218</y>
     <width>34</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/ZcloudDesk/image/coin_tips.png);
color: rgb(255, 255, 255);
font: 10px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p style='line-height:15px'&gt;+60&lt;/p&gt;&lt;p style='line-height:-5px'&gt;云币&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>174</x>
     <y>140</y>
     <width>112</width>
     <height>14</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>112</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>14</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>本月累计签到天数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_line3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>90</width>
     <height>1</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>1</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>1</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(229, 237, 240);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>366</x>
     <y>63</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>14</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>14</width>
     <height>14</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/ZcloudDesk/image/tip.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_round3">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>260</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>21</width>
     <height>21</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>21</width>
     <height>21</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/ZcloudDesk/image/round.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>298</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(51, 51, 51);
font: 14px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>20天</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_line2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>90</width>
     <height>1</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>1</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>1</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(229, 237, 240);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelDay">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>62</y>
     <width>81</width>
     <height>57</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>57</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>57</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 72px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>5</y>
     <width>32</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>28</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>28</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/ZcloudDesk/image/white_close.png);border-radius:none;background-color: rgba(255, 255, 255, 0);}
QPushButton:hover,pressed{border-image: url(:/ZcloudDesk/image/white_close_sel.png);};</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_line1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>90</width>
     <height>1</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>1</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>1</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(229, 237, 240);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="signInButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>338</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2, 164, 253, 1), stop:1 rgba(31, 139, 237, 1));border-radius:4px;font: 18px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);padding-left:8px;}
QPushButton:hover,pressed{ background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(40, 178, 253, 1),stop:1 rgba(64, 156, 240, 1));
		  border-radius:4px;font: 18px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}
QPushButton:disabled{ background-color:rgba(200,200,200,1);border-radius:4px;};</string>
   </property>
   <property name="text">
    <string>立即签到</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>218</y>
     <width>34</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/ZcloudDesk/image/coin_tips.png);
color: rgb(255, 255, 255);
font: 10px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p style='line-height:15px'&gt;+10&lt;/p&gt;&lt;p style='line-height:-5px'&gt;云币&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_line4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>120</width>
     <height>1</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>1</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>1</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(229, 237, 240);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ZcloudDesk.qrc"/>
 </resources>
 <connections/>
</ui>
